fs/localfs/shallowentry.go

```diff
@@ -0,0 +1,30 @@
+package localfs
+
+import (
+	"context"
+	"strings"
+
+	"github.com/kopia/kopia/snapshot"
+)
+
+// Reserved suffixes to identify shallow placeholders for directories and
+// files respectively.
+const (
+	SHALLOWDIRSUFFIX  = ".kopiadir"
```

Comment:
It would make the code simpler to have one suffix. I note in passing that different suffixes have the benefit of making it easier to identity directories vs files in `ls` output.

Reply:
Done.

Reply:
Done in d1b4e4b